static/css/style.css

```diff
@@ -34,6 +34,17 @@ body {
 		padding-right: 20px;
 	}
 }
+.CClogo{
+	height: 40px;
+	width: 40px;
+	position: relative;
+	margin: 0px;
+	margin-right: 30px;
+	margin-left: 10px;
```

Comment:
Either write all in long notation, so margin-top, margin-right, margin-bottom and margin-left or write it in shorthand. Don't mix both :)